cibuildwheel/util.py

```diff
@@ -103,5 +107,39 @@ def get_for_python_version(self, version):
             return self.base_file_path
 
 
+BuildOptions = NamedTuple("BuildOptions", [
+    ("project_dir", str),
+    ("output_dir", str),
+    ("test_command", Optional[str]),
+    ("test_requires", List[str]),
+    ("test_extras", str),
+    ("before_build", Optional[str]),
+    ("build_verbosity", int),
+    ("build_selector", BuildSelector),
+    ("repair_command", str),
+    ("environment", ParsedEnvironment),
+    ("before_test", str),
+    ("dependency_constraints", Optional[DependencyConstraints]),
+    ("manylinux_images", Optional[Dict[str, str]]),
+])
+
+"""
+when drop support of python 3.5 execution.
+class BuildOptions(NamedTuple):
+    project_dir: str
+    output_dir: str
+    test_command: Optional[str]
+    test_requires: List[str]
+    test_extras: str
+    before_build: Optional[str]
+    build_verbosity: int
+    build_selector: BuildSelector
+    repair_command: str
+    environment: ParsedEnvironment
+    before_test: str
+    dependency_constraints: Optional[DependencyConstraints]
+    manylinux_images: Optional[Dict[str, str]]
```

Comment:
Remove this comment to prevent the code and the comment getting out of sync
```suggestion
Replace this definition with a class-style NamedTuple in the
PEP526 style when Python 3.5 host support is dropped
```